frontend/public/components/monitoring.tsx

```diff
@@ -1834,31 +1836,37 @@ type Silences = {
   loadError?: string;
 };
 
-type PrometheusAlert = {
+export type PrometheusAlert = {
   activeAt?: string;
   annotations: PrometheusLabels;
   labels: PrometheusLabels & {
     alertname: string;
   };
-  state: AlertStates;
-  value?: number;
+  state: AlertStates | string;
+  value?: number | string;
+  partialResponseStrategy?: string;
```

Comment:
Let's remove any fields like `partialResponseStrategy` that we are not actually using.